features/facebookImport/src/locales/de/report.json

```diff
@@ -8,5 +8,13 @@
     "sendLater":
         "Nicht senden",
     "send":
-        "Helfen und senden"
+        "Helfen und senden",
+    "status": 
+        "Import Status",
+    "status.details": 
+        "Diese {{number_importers}} Importer haben Ihre Daten eingelesen.",
```

Comment:
This looks shorter than the English equivalent, but OK.

Reply:
I know this is a late comment, since this PR is already merged, but just to clarify. This has already been discussed in the Jira ticket ( https://jira.polypoly.eu/browse/PROD4POD-1635 ), the shorter German version was requested by Sven in the comments.